packages/blade/ios/blade.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why do we need this file?

Reply:
`ios/` files got auto generated on running storybook. Will try deleting and running the command again.

Reply:
Strangely, after I deleted these files and reinstalled app they weren't generated again.